src/lib/native.ts

```diff
@@ -242,12 +242,14 @@ export async function firstinpath(cmdarray) {
     return cmd
 }
 
-export async function editor(file: string, content?: string) {
+export async function editor(file: string, line: number, col: number, content?: string) {
```

Comment:
Is this backwards compatible with old versions of Tridactyl? Easy fix would be to add line? col? args after content.

Reply:
Yes, it's backward compatible. This function is our internal implementation, the excmd remains unchanged.

Reply:
Oh, wow. It was early in the morning, I saw native, and assumed it was in the python script...